yt_dlp/extractor/lsm.py

```diff
@@ -226,9 +228,9 @@ def _real_extract(self, url):
 
 
 class LSMReplayIE(InfoExtractor):
-    _VALID_URL = r'https?://replay\.lsm\.lv/[^/?#]+/(?:ieraksts|statja)/[^/?#]+/(?P<id>\d+)'
+    _VALID_URL = r'https?://replay\.lsm\.lv/[^/?#]+/(?:skaties|klausies)/(?:ieraksts|statja)/[^/?#]+/(?P<id>\d+)'
```

Comment:
The other PR made this new path segment optional. Does the old URL format redirect to the new one? Or please explain